### PR TITLE
General small style changes

### PR DESCRIPTION
**Pull request by SmithPeder**
There have always been small CSS things that bug me every day. This fixes some of them.

I know the new frontpage will eventually fix this, but I just need something temp

### Festet Oppslag content spacing was off
image
image

### Articles are too big, and people don't write that much "description", and again the content spacing was off
image
image

### It annoyed me that the bottom of the articles and events didn't line up
image
image

### Title [Bedpress og Kurs] was not indented like all the other headers
image
image

### Saving an article was dumb
image
image

### CODE REVIEW

File: app/components/Reactions/assets/AddReactionEmoji.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const AddReactionEmoji = ({ color }: Props) => (
   <div className={styles.container}>
     <span>
       <svg
-        ariaHidden="true"
+        aria-hidden="true"
```
Comment (SmithPeder) [inline app/components/Reactions/assets/AddReactionEmoji.js:13]: and this Warning annoyed the shit out me

Comment (LudvigHz) [reply app/components/Reactions/assets/AddReactionEmoji.js:13]: LOL yeah, this one's a goodamn bi***

File: app/components/Button/Button.css [hunk 2]
```diff
@@ -89,3 +89,8 @@
   border: none;
   cursor: pointer;
 }
+
+.danger {
+  background: var(--color-red-3);
+  color: var(--color-white);
+}
```
Comment (LudvigHz) [inline app/components/Button/Button.css:96]: Don't we already have a "delete" button styling?

File: app/routes/overview/components/CompactEvents.css [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 .compactEvents h3 {
   margin-bottom: 10px;
-  padding: 0;
+  padding: 0 10px;
```
Comment (LudvigHz) [inline app/routes/overview/components/CompactEvents.css:13]: I'm not so sure about this. I feel like the old style is better. The indented title looks kinda "off" to me :thinking:

**Review by LudvigHz — APPROVED** [review]
Nice :100: Always nice to fix the small, annoying things.

File: app/components/Button/Button.css (hunk 2 above)
Comment (SmithPeder) [reply app/components/Button/Button.css:96]: We had a danger prop, that did nothing 🤷🏻‍♂️

Comment (SmithPeder) [reply app/components/Button/Button.css:96]: Ahh, wait, the "dark" is red... So dark is actually "danger/delete"
That's...

File: app/routes/overview/components/CompactEvents.css (hunk 3 above)
Comment (SmithPeder) [reply app/routes/overview/components/CompactEvents.css:13]: I was going back and forth for like 15min, haha. Both look kinda "off", but I feel this looks "less" off somehow. But I'm open for a vote

Comment (erlingfn) [reply app/routes/overview/components/CompactEvents.css:13]: I agree with @LudvigHz on that it looks a bit off, but perhaps we should be consistent on the  headers?

**Review by ivarnakken — APPROVED** [review]
✨✨

File: app/routes/overview/components/CompactEvents.css (hunk 3 above)
Comment (juniwbjerde) [reply app/routes/overview/components/CompactEvents.css:13]: I don't really care if they're indented or not, but they should for sure be the same as the rest of the headers. So if we go back to the old style we should change all the other header to fit that styling too

**Review by juniwbjerde — APPROVED** [review]

File: app/routes/overview/components/CompactEvents.css (hunk 3 above)
Comment (eikhr) [reply app/routes/overview/components/CompactEvents.css:13]: I agree that it looks a little off, but I think it looks better than it being inconsistent

**Review by eikhr — APPROVED** [review]

**Pull request merged**